tools/network/telemetryURIUpdateService.go

```diff
@@ -41,9 +41,9 @@ func lookupTelemetryEndpoint(cfg config.Local, genesisNetwork protocol.NetworkID
 	for _, bootstrapID := range bootstrapArray {
 		addrs, err := ReadFromSRV("telemetry", bootstrapID, cfg.FallbackDNSResolverAddress)
 		if err != nil {
-			log.Warnf("An issue occurred reading telemetry entry for: %s", bootstrapID)
+			log.Infof("An issue occurred reading telemetry entry for: %s", bootstrapID)
```

Comment:
nit:
1. when formatting input strings, make sure to place them in quotes so we could identify white space issues. i.e. '%s'
2. If there is untyped, non-nil error, you want to log the error string as well.